docs/src/components/Story.tsx

```diff
@@ -35,6 +35,7 @@ export const Story = ({ path, ...props }: StoryProps) => {
                         <MuiThemeProvider theme={theme}>
                             <CodeBlock
                                 language="tsx"
+                                // @ts-expect-error live prop is missing in the types
```

Comment:
I also stumbled across this last week, i also can't find any documentation on this `live` prop. Do you know if this is even needed?

Reply:
This is for the interactive code editor: https://docusaurus.io/docs/markdown-features/code-blocks#interactive-code-editor.

We need this for interactive stories, e.g., [here](https://docs.comet-dxp.com/docs/admin-components/components/FieldSet#example).